packages/params/src/setPreset.ts

```diff
@@ -38,3 +39,14 @@ console.log({SLOTS_PER_EPOCH})
 
   userSelectedPreset = presetName;
 }
+
+export let userOverrides: Partial<BeaconPreset> | null = null;
+
+/**
+ * @param presetName - the preset to use as a base
+ * @param overrides - customized fields
+ */
+export function setCustomPreset(presetName: PresetName, overrides: Partial<BeaconPreset>): void {
+  setActivePreset(presetName);
+  userOverrides = overrides;
+}
```

Comment:
Why not extend setActivePreset() function above?

Reply:
good idea. and make `overrides` optional.